fuse_viz/package.xml

```diff
@@ -23,6 +23,8 @@
   <depend>rviz</depend>
   <depend>tf2_geometry_msgs</depend>
 
+  <build_depend>roslint</build_depend>
```

Comment:
In other packages this is a `test_depend`, e.g. https://github.com/locusrobotics/fuse/blob/devel/fuse_graphs/package.xml#L22
```suggestion
  <test_depend>roslint</test_depend>
```

Reply:
Haha, I just grabbed a random package to copy it from, and I chose `fuse_models`:

https://github.com/locusrobotics/fuse/blob/devel/fuse_models/package.xml#L38

But that's because I have the `roslint` stanza outside testing in that package.